python/lsst/atmospec/utils.py

```diff
@@ -628,10 +636,9 @@ def makeQuery(dataId):
                        "instrument='LATISS'")
 
         return queryString
-    repo = "LATISS" if not embargo else "/repo/embargo"
 
     # TODO: use LATISS_DEFAULT_COLLECTIONS here?
-    inputs = ['LATISS/raw/all', 'refcats', 'LATISS/calib']
+    inputs = ['LATISS/raw/all', 'refcats', 'LATISS/calib', 'LATISS/calib/legacy']
```

Comment:
Is that collection really available with all repos?

Reply:
Hopefully (maybe already?) `embargo_old` disappears. I think you're the only person who uses this function (and me, sometimes) so if you're OK with this not always working, then again, I'm OK with it for now, but it's definitely going to need changing in the future.